tests/bugfixes/github/test_issue_283.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+import os.path
+
+class TestFirstPoC(metaclass=system_tests.CaseMeta):
+    """
+    Regression test for the bug described in:
+    https://github.com/Exiv2/exiv2/issues/283
+    """
+    url = "https://github.com/Exiv2/exiv2/issues/283"
+
+    def check_no_ASAN_UBSAN_errors(self, i, command, got_stderr, expected_stderr):
```

Comment:
This won't work, you have to override `compare_stderr` (that's the function that performs the checks) to call `check_no_ASAN_UBSAN_errors`. Overriding the later will not do anything, as it isn't called by default.

Sorry, should have been clearer about that.

Reply:
Ah, forget what I said. I didn't look further down where you set `compare_stderr` to `check_no_ASAN_UBSAN_errors`. This is of course fine then.

Reply:
Yes, I was wondering the best way to do it. Of course, I tried to run this before I submitted the changes and it works :wink: